README.md

```diff
@@ -6,3 +6,23 @@ Help wanted!
 
 Channel Invite:
 https://bestow-regional.api.smartthings.com/invite/eGM6WwPwP2AP 
+
+After accepting this channel invite and enrolling, any HomeSeer switches you add should automatically be assigned this Edge Driver.
+
+To see if a switch is using this device handler, go to Devices, open the switch and click the "..." button. If "Driver" is in the list, then it's using the new Edge DTH code. The driver name will be "HomeSeer Z-Wave Switch" for this DTH. If "Driver" is not present, it's using the legacy Groovy handler. In this case you should remove the device and re-add it.
+
+## Removing the old handler
+
+When removing the device via the SmartThings app, it will prompt you to follow your manufacturer's directions to unpair the old device. For all models the factory reset sequence is:
```

Comment:
```suggestion
When removing the device via the SmartThings app, it will prompt you to follow your manufacturer's directions to exclude the old device. Once the exclusion process is started by the app, single click and release the rocker switch.
```
http://manuals-backend.z-wave.info/make.php?lang=en&sku=HS-WD100+&cert=ZC10-16055087

Reply:
Factory resetting shouldn't be necessary, so I suggest removing those instructions.